styles/main.css

```diff
@@ -1,16 +1,21 @@
 body {
```

Comment:
I think adding some space between the sections would make it more reader-friendly. Maybe we could make a new style rule for `section` and add `margin-top: 60px;`